File: V3/ServerApp/src/main/java/lk/earth/earthuniversity/report/entity/FenceReport.java
package lk.earth.earthuniversity.report.entity;

public class FenceReport {

    private Integer id;
    private String fencetype;
    private Long count;
    private double percentage;

    public FenceReport() { }

    public FenceReport(String fencetype, Long count) {
        this.fencetype = fencetype;
        this.count = count;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getFencetype() {
        return fencetype;
    }

    public void setFencetype(String fencetype) {
        this.fencetype = fencetype;
    }

    public Long getCount() {
        return count;
    }

    public void setCount(Long count) {
        this.count = count;
    }

    public double getPercentage() {
        return percentage;
    }

    public void setPercentage(double percentage) {
        this.percentage = percentage;
    }
}

File: V3/ClientApp/src/app/report/entity/fencereport.ts
export class fencereport {
  public id!: number;
  public fencetype!: string;
  public count!: number;
  public percentage!: number;

  constructor(id: number, fencetype: string, count: number, percentage: number) {
    this.id = id;
    this.fencetype = fencetype;
    this.count = count;
    this.percentage = percentage;
  }
}
